tasks/plugins.yml

```diff
@@ -19,3 +31,10 @@
   with_items:
     - graylog
     - default-groups
+
+- name: install ldap plugin
+  template:
+    src: pritunl-plugin-ldap.j2
+    dest: /var/lib/pritunl/plugins/ldap.py
+  notify: restart pritunl
+  when: pritunl_plugins | selectattr("name","equalto","ldap") | list | count == 1 and pritunl_api is defined
```

Comment:
@raphapr what about use callback plugin `json_query` and use "()" in the expression to best fit your code? =]

Reply:
@mtulio I wish that. but ```json_query``` is ansible 2.2+  ):

Reply:
Let's migrate to ansible 2.3 :dancer: